API/files/README.md

```diff
@@ -28,7 +28,8 @@ If no `content` is passed, then the path is treated as an empty directory
 ```js
 {
   path: '/tmp/myfile.txt',
-  node: DAGNode
+  hash: 'QmHash', // base58 encoded multihash
```

Comment:
CIDs will allow for different versions of encoding, i.e. non base58. This interface though binds itself to using base58 which I find the most concerning part.